packages/ERTP/src/amountMath.js

```diff
@@ -170,7 +170,7 @@ const coerceLR = (h, leftAmount, rightAmount) => {
 };
 
 /** @type {AmountMath} */
```

Comment:
Doesn't the type's name need to change in order to not conflict with the export?

Reply:
Nope, I think the types are a different name space